charts/backend/templates/secrets.yaml

```
@@ -55,8 +85,36 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: entitlements-secrets
+  labels:
+    app: {{ .Values.name }}
+    chart: {{ template "backend.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}  
 type: Opaque
 stringData:
   POSTGRES_PASSWORD: {{ .Values.secrets.postgres.dbPassword }}
   OIDC_CLIENT_SECRET: {{ .Values.secrets.oidcClientSecret }}
-
+---
+{{- if not .Values.secrets.kas.external }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.kas.secretRef.name }}
+  labels:
+    app: {{ .Values.name }}
```

```suggestion
    app: {{ .Values.name }}
```

`app:` should be the actual app this secret belongs to (kas, entitlements, etc)

`release:` and `chart:` are for a higher-level tag.

This is easier to do/more obvious if we move each secret down to the subchart it belongs with.